src/shared/state/state.h

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef ADDONSTATE_H
+#define ADDONSTATE_H
+
+#include <QJsonObject>
```

Comment:
you don't need this.

Reply:
Are you sure? I get errors when I remove it. Note this file doesn't have a corresponding .cpp file. Should I add one?